src/main/resources/application.conf

```diff
@@ -46,9 +46,18 @@ backend-for-frontend {
 
   storage {
     kind = "S3"
-    container = ${STORAGE_CONTAINER}
-    consumer-documents-path = "agreement/consumer-docs"
-    consumer-documents-path = ${?CONSUMER_DOCS_PATH}
+    consumer-documents {
+        container = ${CONSUMER_DOCS_CONTAINER}
+        path = "agreement/consumer-docs"
+        path = ${?CONSUMER_DOCS_PATH}
+    }
+    allow-list {
+        container = ${ALLOW_LIST_CONTAINER}
+        path = "allowlist"
+        path = ${?ALLOW_LIST_PATH}
+        filename = "allowlist.txt"
+        filename = ${?ALLOW_LIST_FILENAME}
```

Comment:
These environment variables may be added to platform deployment:
- `ALLOW_LIST_CONTAINER`
- `ALLOW_LIST_PATH`
- `ALLOW_LIST_FILENAME`

And this one should be renamed:
- `STORAGE_CONTAINER` to `CONSUMER_DOCS_CONTAINER`

These changes should be made in the [platform deployment repo](https://github.com/pagopa/pdnd-interop-platform-deployment)